src/gui/editors/PianoRoll.cpp

```diff
@@ -1740,42 +1687,42 @@ void PianoRoll::mousePressEvent(QMouseEvent * me )
 					// +32 to quanitize the note correctly when placing notes with
 					// the mouse.  We do this here instead of in note.quantized
 					// because live notes should still be quantized at the half.
-					TimePos note_pos( pos_ticks - ( quantization() / 2 ) );
-					TimePos note_len( newNoteLen() );
+					TimePos note_pos(pos_ticks - (quantization() / 2));
+					TimePos note_len(newNoteLen());
 
-					Note new_note( note_len, note_pos, key_num );
-					new_note.setSelected( true );
-					new_note.setPanning( m_lastNotePanning );
-					new_note.setVolume( m_lastNoteVolume );
-					created_new_note = m_pattern->addNote( new_note );
+					Note new_note(note_len, note_pos, key_num);
+					new_note.setSelected(true);
+					new_note.setPanning(m_lastNotePanning);
+					new_note.setVolume(m_lastNoteVolume);
+					created_new_note = m_pattern->addNote(new_note);
 
-					const InstrumentFunctionNoteStacking::Chord & chord = InstrumentFunctionNoteStacking::ChordTable::getInstance()
-						.getChordByName( m_chordModel.currentText() );
+					const InstrumentFunctionNoteStacking::Chord &chord = InstrumentFunctionNoteStacking::ChordTable::getInstance()
+																			 .getChordByName(m_chordModel.currentText());
```

Comment:
This comes from `ColumnLimit: 0`, which means "Respect human line break decisions", which we used to keep our CTOR initializer lists untouched (one initializer per row). In this case, the line break decision of the human was respected, but the `.getChordByName` must be indented, which results in this mess. I'm rather for using a `ColumntLimit: 120` and let the CTOR initializer lists break before the comma:

```
classname::classname()
    : obj1(x1)
    , obj2(x2)
    , obj3(x3)
```

Then, the `ColumnLimit: 120` could be used and this line that you marked here gets indented in a sane way.

Reply:
We might have to just accept some other changes to format. 🤔

Reply:
> I'm rather for using a ColumntLimit: 120 and let the CTOR initializer lists break before the comma

I agree. It may not win a beauty contest, but it is readable, and it apparently works better in the context of fully automatic formatting, which seems kinda important if we want to forever get rid of manual code-style reviews.

Reply:
OK, I can try a re-run with `ColumnLimit: 120` and letting CTOR initializer lists break before comma. We asked in Discord and 4 users agreed, so I guess it will be OK.

Reply:
With `ColumnLimit: 0`:

```
					const InstrumentFunctionNoteStacking::Chord& chord =
						InstrumentFunctionNoteStacking::ChordTable::getInstance().getChordByName(
							m_chordModel.currentText());
```

OK ?

Reply:
I've actually remedied this a little in my WIP refactor PR by moving the `InstrumentFunctionNoteStacking::ChordTable::getInstance()` into its own variable for reuse in most functions.

But to answer yeah that's good 👍 😄